### PR TITLE
add export to bq feature

### PR DESCRIPTION
**Pull request by lvvvvvf**
Sorry I did not notice the comments before

### CODE REVIEW

**Comment by lvvvvvf** [comment]
Anything needed from my side ?  thanks

File: asset/cloud-client/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 Google LLC
```
Comment (lesv) [inline asset/cloud-client/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java]: Should this be 2020?

**Comment by lesv** [comment]
PING - I think this needed a change to copyright, are you abandoning this?

@stephaniewang526 Can you take a look and figure out if we should close this?

File: asset/cloud-client/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java (hunk 1 above)
Comment (lvvvvvf) [reply asset/cloud-client/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java]: Done, thx

**Comment by lesv** [comment]
Needs a bit of work.
```
------------------------------------------------------------
- testing asset/cloud-client
------------------------------------------------------------
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 91.578 s <<< FAILURE! - in com.example.asset.QuickStartIT
[ERROR] testBatchGetAssetsHistory(com.example.asset.QuickStartIT)  Time elapsed: 2.615 s  <<< ERROR!
com.google.api.gax.rpc.InvalidArgumentException: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Request contains an invalid argument.
Caused by: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Request contains an invalid argument.

[ERROR] Errors:
[ERROR]   QuickStartIT.testBatchGetAssetsHistory » InvalidArgument io.grpc.StatusRuntime...
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0
```

**Comment by lvvvvvf** [comment]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.example.asset.QuickStartIT
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 59.087 s - in com.example.asset.QuickStartIT
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] 
[INFO] --- maven-failsafe-plugin:3.0.0-M3:verify (default) @ asset-google-cloud-samples ---
[INFO] 
[INFO] --- maven-checkstyle-plugin:3.0.0:check (default) @ asset-google-cloud-samples ---
[INFO] Starting audit...
Audit done.
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ asset-google-cloud-samples ---
[INFO] Deleting /usr/local/google/home/lvv/java-docs-samples/asset/cloud-client/target
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:03 min
[INFO] Finished at: 2020-04-01T19:52:38-07:00
[INFO] ------------------------------------------------------------------------


it works locally.  Do I miss anything?

**Comment by lesv** [comment]
Should be all you need.  Once Kokoro passes - your good to go - not sure why it's not already done.

**Comment by lesv** [comment]
#Java 8
```
------------------------------------------------------------
- testing asset/cloud-client
------------------------------------------------------------
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 194.724 s <<< FAILURE! - in com.example.asset.QuickStartIT
[ERROR] testBatchGetAssetsHistory(com.example.asset.QuickStartIT)  Time elapsed: 3.288 s  <<< ERROR!
com.google.api.gax.rpc.InvalidArgumentException: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Request contains an invalid argument.
Caused by: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Request contains an invalid argument.
```
You might wish to check w/ a real Java 8 compiler.

**Pull request merged**